src/components/Modal/styled.tsx

```diff
@@ -122,3 +123,7 @@ export const Content = styled.div`
   padding: ${({ theme }) => em(theme.honeycomb.size.increased)};
   min-height: 100%;
 `;
+
+export const LoadingState = styled.div`
+  margin-right: ${({ theme }) => em(theme.honeycomb.size.small)};
```

Comment:
If you put this inside `<Title />`, you need to take into account the `font-size` of that component.

```suggestion
  margin-right: ${({ theme }) => em(theme.honeycomb.size.small, theme.honeycomb.size.reduced)};
  
  @media (${bigScreen}) {
    margin-right: ${({ theme }) => em(theme.honeycomb.size.small, theme.honeycomb.size.normal)};
  }
```